### PR TITLE
improves style explanaiton

### PR DESCRIPTION
**Pull request by Nowosad**

### CODE REVIEW

**Comment by Robinlovelace** [comment]
Will take a look now @Nowosad

File: 09-mapping.Rmd [hunk 1]
```diff
@@ -316,9 +316,9 @@ tm_shape(nz) + tm_symbols(size = "Land_area", size.scale = tm_scale_intervals())
 \index{tmap (package)!interval scale}
 The `tm_scale_intervals()` function splits the input data values into a set of intervals.
 In addition to manually setting `breaks`, **tmap** allows users to specify algorithms to create breaks with the `style` argument automatically.
-Here are some of the most useful scale functions (Figure \@ref(fig:break-styles)):
+The default is `tm_scale_intervals(style = "pretty")`, which rounds breaks into whole numbers where possible and spaces them evenly.
+Here, we highlight other useful scales (Figure \@ref(fig:break-styles)):
```
Comment (Robinlovelace) [inline 09-mapping.Rmd:320]:
```suggestion
Other options are listed below and presented in Figure \@ref(fig:break-styles).
```

Bit more concise and clear, not totally clear what the 'Here' is referring to.

**Review by Robinlovelace — APPROVED** [review]

**Comment by Nowosad** [comment]
(Relates to https://github.com/geocompx/geocompr/issues/1069)

**Review by tibbles-and-tribbles — COMMENTED** [review]
Looks good

**Pull request merged**